data_collection/advertisements/powerblade_adv_influx.js

```diff
@@ -0,0 +1,246 @@
+#!/usr/bin/env node
+// original file: https://github.com/lab11/powerblade/tree/master/data_collection/advertisements
+
+var noble = require('noble');
+var config = require('./config');
+var influx;
+
+function getDateTime() {
+
+    var date = new Date();
```

Comment:
This probably works fine, so no point changing it now, but in the future you might want to use libraries such as [Moment](http://momentjs.com/docs/#/displaying/) to do this kind of thing with a format string.

Reply:
Thanks, I did not know about that library.